wrote_about_us/Leadership_personality profiles.json

```
@@ -0,0 +1,24 @@
+{
+  "best_articles": [
+    {
+      "date": "2023-17-23",
+      "title": "Od podprsenek k záchraně životů. Zakladatel Mild Blue pracuje s daty a umělou inteligencí, ale byznys řídí vlastním rozumem",
+      "url": "https://sj.news/od-podprsenek-k-zachrane-lidskych-zivotu-zakladatel-mild-blue-pracuje-s-daty-a-umelou-inteligenci-ale-byznys-ridi-vlastnim-rozumem/",
+      "img": "sj.news.webp",
+      "source": "SJ News",
+      "tags": [
+        "article",
+        "leadership"
+      ]
+    }
+  ],
+  "urls": [
+    "https://sj.news/od-podprsenek-k-zachrane-lidskych-zivotu-zakladatel-mild-blue-pracuje-s-daty-a-umelou-inteligenci-ale-byznys-ridi-vlastnim-rozumem/",
```

```suggestion
    "https://sj.news/od-podprsenek-k-zachrane-lidskych-zivotu-zakladatel-mild-blue-pracuje-s-daty-a-umelou-inteligenci-ale-byznys-ridi-vlastnim-rozumem/"
```
Tu je naopak čiarka na konci navyše

---

opraveno